programs/virtual-curve/src/instructions/migration/meteora_damm/meteora_damm_claim_lp_token.rs

```diff
@@ -98,6 +104,13 @@ pub fn handle_migrate_meteora_damm_partner_claim_lp_token<'info>(
 pub fn handle_migrate_meteora_damm_creator_claim_lp_token<'info>(
     ctx: Context<'_, '_, '_, 'info, MigrateMeteoraDammClaimLpTokenCtx<'info>>,
 ) -> Result<()> {
+    let virtual_pool = ctx.accounts.virtual_pool.load()?;
+
+    require!(
```

Comment:
^Ditto for possible backward compatibility

Reply:
Ditto^